### PR TITLE
fix(models): replace should with must

### PR DESCRIPTION
**Pull request by WikiRik**
cc @dilyanpalauzov

### CODE REVIEW

File: models/Circle.js [hunk 1]
```diff
@@ -6,22 +6,22 @@ const Circle = sequelize.define('circle', {
         allowNull: false,
         defaultValue: '',
```
Comment (dilyanpalauzov) [inline models/Circle.js:7]: The defaultValue is the empty string, which is invalid value.

Comment (WikiRik) [reply models/Circle.js:7]: Yeah, I just replaced it as a first step. But I'll go through the defaultValues as well now. Thanks for the comment!

Comment (WikiRik) [reply models/Circle.js]: Fixed

File: models/Body.js [hunk 2]
```diff
@@ -17,8 +17,7 @@ const Body = sequelize.define('body', {
     },
     task_description: {
         type: Sequelize.TEXT,
-        allowNull: true,
-        defaultValue: ''
+        allowNull: true
```
Comment (dilyanpalauzov) [inline models/Body.js:20]: In this example, what is the difference between task_description being the empty string and task_description being NULL?

Compare to the `address` field, which can be NULL, but cannot be the empty string.  For the record, the address of [AEGEE-Samara](https://my.aegee.eu/bodies/143) is “we don't have that position”.

**Pull request closed without merging**